App/util/index.ts

```diff
@@ -0,0 +1,76 @@
+import AsyncStorage from '@react-native-async-storage/async-storage';
+import {MediaArr, mediaObj} from '../@types';
+
+const SetStorage = async (key: string, value: string[]) => {
+  try {
+    await AsyncStorage.setItem(key, JSON.stringify(value));
+  } catch (error) {
+    console.log('Error Saving Data');
+  }
+};
+
+const GetStorage = async (key: string) => {
+  try {
+    const value = await AsyncStorage.getItem(key);
+    if (value !== null) {
+      return JSON.parse(value);
+    }
+  } catch (error) {
+    console.log('Error Retrieving Data');
+    return;
+  }
+};
+
+const RemoveItem = async (key: string) => {
+  try {
+    AsyncStorage.removeItem(key);
+  } catch (error) {
+    console.log('Error Removing Item', error);
+  }
+};
+
+const getImage = (obj: MediaArr, id: string) => {
```

Comment:
rename obj to proper name Like medioObjectList or array, and they Type MediaArr should be MediaObj[ ].